lib/plugins/create/templates/google-go/serverless.yml

```diff
@@ -0,0 +1,44 @@
+service: gcf-go111 # NOTE: Don't put the word "google" in here
+
+provider:
+  name: google
+  runtime: go111
+  project: my-project
+  # the path to the credentials file needs to be absolute
```

Comment:
Would add a reference to the docs as we have in the `examples` repo:

```

# The GCF credentials can be a little tricky to set up. Luckily we've documented this for you here:
# https://serverless.com/framework/docs/providers/google/guide/credentials/
# 
# the path to the credentials file needs to be absolute
```

Reply:
Great docs! I also added to the node one.